src/transaction/txParser.ts

```diff
@@ -148,14 +160,37 @@ const parseRelay = (poolRelay: any): _PoolRelay => {
 }
 
 const parseTxCerts = (txCertificates: any[]): _Certificate[] => {
+  const parseStakeCredentials = (stakeCredentials: any): StakeCredentials => {
```

Comment:
I don't understand why plural is used here, and also in the type StakeCredentials --- it's just a single credential per certificate.

Reply:
Didn't implement this originally, but also noticed that it was a bit weird. Changed it to singular.

Reply:
Didn't originally implement this, but also noticed that it was a bit weird. Changed it to singular.